### ISSUE TITLE
buttons not responding to emulated touch events in chrome devtools 

### ISSUE THREAD

**Issue by jamesjsewell** [photonstorm/phaser-ce#498]
I'm not sure why this is, pointer touch events seem to be working ok but not for touch events on buttons.

**Comment by samme** [comment]
Check the pointer is placed correctly in <https://samme.github.io/phaser-examples-mirror/debug/debug%20input.html>?

**Comment by jamesjsewell** [comment]
@samme this is for CE. the buttons do work properly on my mobile phone but not in chrome devtools. chrome dev tools allows you to enter an emulated mobile device for testing and normally html5 buttons work in that mode. Are there any testing environments you suggest i use for mobile?

**Comment by samme** [comment]
<https://samme.github.io/phaser-examples-mirror/buttons/action%20on%20click.html> is working for me in Chrome's mobile simulator.

**Comment by jamesjsewell** [comment]
@samme am I using the correct thing? this isn't working [![https://gyazo.com/20859d6ed22b0e994efeed21614b4141](https://i.gyazo.com/20859d6ed22b0e994efeed21614b4141.png)](https://gyazo.com/20859d6ed22b0e994efeed21614b4141)

**Comment by samme** [comment]
That's the same thing I'm using. Is yours Chrome 65? Did you check <https://samme.github.io/phaser-examples-mirror/debug/debug%20input.html>?

**Comment by jamesjsewell** [comment]
@samme that does work though it is buggy for different devices, like when i choose the ipad for example, the dot is offset from where i clicked. The dots offset is sometimes non existent, sometimes small, and like on the ipad large.

**Comment by samme** [comment]
I think I saw that problem before. Make sure your Chrome is updated to the latest.

**Comment by jamesjsewell** [comment]
awesome that did the trick. I went to the "more" button on the top right of google chrome, the three dots. went down to the 'help' option in the popup menu. chose 'about google chrome' in the submenu. There I saw that my chrome auto updates and that I had the latest update installed, it said i just needed to relaunch chrome. guess i haven't closed chrome in a while

Thanks @samme ! this is really handy to have working obviously
